google/cloud/spanner/value.h

```diff
@@ -314,6 +314,18 @@ class Value {
    */
   friend void PrintTo(Value const& v, std::ostream* os);
 
+  /**
+   * Outputs string representation of a given Value to the provided stream.
```

Comment:
I think this should copy the warning from the `PrintTo` function:

```
@warning This is intended for debugging and human consumption only, not
    machine consumption as the output format may change without notice.
```

Reply:
Done.